Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -49,6 +50,37 @@ public struct Evaluation {
     public let variableKey: VariableKey?
     public let variableValue: VariableValue?
     public let variableSchema: VariableSchema?
+
+    public init(
+            featureKey: FeatureKey,
+            reason: EvaluationReason,
+            bucketValue: BucketValue? = nil,
+            ruleKey: RuleKey? = nil,
+            error: Error? = nil,
+            enabled: Bool? = nil,
+            traffic: Traffic? = nil,
+            sticky: OverrideFeature? = nil,
+            initial: OverrideFeature? = nil,
+            variation: Variation? = nil,
+            variationValue: VariationValue? = nil,
+            variableKey: VariableKey? = nil,
+            variableValue: VariableValue? = nil,
+            variableSchema: VariableSchema? = nil) {
+        self.featureKey = featureKey
+        self.reason = reason
+        self.bucketValue = bucketValue
+        self.ruleKey = ruleKey
+        self.error = error
+        self.enabled = enabled
+        self.traffic = traffic
+        self.sticky = sticky
+        self.initial = initial
+        self.variation = variation
+        self.variationValue = variationValue
+        self.variableKey = variableKey
+        self.variableValue = variableValue
+        self.variableSchema = variableSchema
```

Comment:
didn't realize it will result into this much work in Swift 🤯

Reply:
As a human, I hate this as well :D. My take is that in Swift there are reminiscences of the verbosity of Objective-C